content/en/docs/concepts/workloads/pods/ephemeral-containers.md

```diff
@@ -81,134 +81,83 @@ you can view processes in other containers.
 ### Examples
 
 {{< note >}}
-The examples in this section require the `EphemeralContainers` [feature
-gate](/docs/reference/command-line-tools-reference/feature-gates/) to be
-enabled, and Kubernetes client and server version v1.16 or later.
+The examples in this section require Kubernetes server components of v1.16
+or later with the `EphemeralContainers` [feature gate](
+/docs/reference/command-line-tools-reference/feature-gates/) enabled and
+`kubectl` version v1.18 or later.
```

Comment:
Because of version skew rules, I wonder if these version numbers are right?

(it might make life easier for readers if this were moved to a task page with `min-kubernetes-server-version:` set in its header)